segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java

```diff
@@ -490,6 +500,11 @@ public void mergeSegments(MergeSegments mergeSegments) {
                                                                                             properties.getLength()));
                                         });
                             return null;
+                        } else if (Exceptions.unwrap(e) instanceof BadAttributeUpdateException) {
+                            log.debug(mergeSegments.getRequestId(), "Conditional merge failed (Source segment={}, Target segment={})",
+                                    mergeSegments.getSource(), mergeSegments.getTarget(), e);
```

Comment:
This is going to print the full stack trace which will pollute the logs. We don't need it for `BadAttributeUpdateException`; just print the message (`e.toString()` - remember to add a `{}` to the message).

Reply:
Done.